frontend/model/contracts/identity.js

```diff
@@ -121,14 +123,14 @@ sbp('chelonia/defineContract', {
     'gi.contracts/identity/createDirectMessage': {
       validate: (data, { state, getters }) => {
         objectOf({
-          privacyLevel: unionOf(...Object.values(CHATROOM_PRIVACY_LEVEL).map(v => literalOf(v))),
+          groupContractID: string,
```

Comment:
Can you add comments explaining the difference between `groupContractID` and `contractID`?

Also, shouldn't `groupContractID` be optional? DMs can be created without referencing a group.

Same goes for the lines below in `joinDirectMessage`